components/server/src/server.rs

```diff
@@ -937,6 +943,83 @@ impl<ER: RaftEngine> TiKVServer<ER> {
                 io_metrics.flush(duration);
             });
     }
+    fn init_storage_stats_task(&self, engines: Engines<RocksEngine, ER>) {
+        let flag = self.write_permission.clone();
+        let config_disk_capacity: u64 = self.config.raft_store.capacity.0;
+        let store_path = self.store_path.clone();
+        let snap_path = self.snap_path.clone();
+        self.background_worker
+            .spawn_interval_task(DEFAULT_STORAGE_STATS_INTERVAL, move || {
+                let disk_stats = match fs2::statvfs(&store_path) {
```

Comment:
Why not reuse code in `components/raftstore/src/store/worker/pd.rs`?

Reply:
My first method is to reuse the logic and introduce a global variable to be updated, but my colleagues do not recommend this style of introducing an additional global variable.

Reply:
first commit reuse pd.rs, but removed for some advice. now, it has been repushed.